scripts/analytics/addon_count.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from website.settings import ADDONS_AVAILABLE
+from website.app import init_app
+from website.settings import KEEN as keen_settings
+from keen.client import KeenClient
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def count():
+    counts = []
+    addons_available = {k: v for k, v in [(addon.short_name, addon) for addon in ADDONS_AVAILABLE]}
+    for short_name, addon in addons_available.iteritems():
+        user_count = addon.settings_models['user'].find().count() if addon.settings_models.get('user') else 0
+        node_count = addon.settings_models['node'].find().count() if addon.settings_models.get('node') else 0
+        counts.append({
+            'provider': short_name,
```

Comment:
Note from @felliott that we could use some more nesting here, with possibly some extra info as well. For example:
```
{
  'provider': {
    ‘id’: 'googledrive’,
  },
  'users': {
    'total': 123,
  },
  'nodes': {
    'total': 240,
    'connected': 123,
    'deleted': 5,
    'unconnected': 112,
  },
  'extra': {
    'dataverse': {
       'hosts': {
          'dataverse.harvard.edu': 20,
          'dataverse.virginia.edu': 10,
          'demo.dataverse.org': 50
        },
    },
    'github': {
       'users': {
         'uniq': 10,
       },
       'repos': {
         'uniq': 15,
       },
       'user_repos': {
         'uniq': 20,
       },
    },
  },
},
```